### PR TITLE
Fix #1524:  Removing previous patch of mozillathimblelivepreview script preloading in homepage  

### LINKED ISSUE #1524: Remove mozillathimblelivepreview script preloading in homepage
Despite the fact that neither of those things should hold up the app loading and starting up a UI.

### PR DESCRIPTION
**Pull request by evantsyferov**
Working on fixing the second part of issue [1524](https://github.com/mozilla/thimble.mozilla.org/issues/1524)

### CODE REVIEW

File: public/homepage/scripts/main.js [hunk 1]
```diff
@@ -1,13 +1,3 @@
-/* global requirejs */
```
Comment (humphd) [inline public/homepage/scripts/main.js]: This comment can't be removed, it's an instruction to our linters about a global variable.

File: public/homepage/scripts/main.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+// global requirejs
```
Comment (humphd) [inline public/homepage/scripts/main.js]: What you had before was right, please undo this change.

Comment (evantsyferov) [inline public/homepage/scripts/main.js]: I tried submitting the code with  /* global requirejs */ but Travis fails the push and stays it is an error since requirejs is declared and not used.

Comment (gideonthomas) [inline public/homepage/scripts/main.js]: @evantsyferov then remove the entire comment.

**Comment by humphd** [comment]
I'm finishing this in https://github.com/mozilla/thimble.mozilla.org/pull/2234
